src/main/web_src/src/components/fagsystem/udistub/form/validation.js

```diff
@@ -41,11 +41,40 @@ const oppholdSammeVilkaar = Yup.object({
 	oppholdstillatelseType: requiredString
 })
 
+const ikkeOppholdSammeVilkaar = Yup.object({
+	avslagEllerBortfall: Yup.object({
+		avgjorelsesDato: Yup.date().required(),
```

Comment:
Ser ikke ut til at valideringen fungerer som den skal, for jeg får gå videre uten å fylle ut avgjorelsesDato og avslagGrunnlagOverig.

Reply:
Whoops, de var egentlig bare med for å teste, tror ingen skal være required egentlig 😅